frontend/src/metabase/browse/components/TableBrowser/TableBrowser.styled.tsx

```diff
@@ -11,6 +11,7 @@ import {
 
 export const TableGrid = styled(Grid)`
   width: 100%;
+  flex: 1;
```

Comment:
should we delete this? did we mean to touch the table browser?

Reply:
Yeah, I don't think we meant to do this